next/test/form.test.ts

```diff
@@ -5,14 +5,4 @@ describe('createHeadlessForm', () => {
   it('should be a function', () => {
     expect(createHeadlessForm).toBeInstanceOf(Function)
   })
-
-  it('returns empty result given an empty schema', () => {
-    const result = createHeadlessForm({}, { initialValues: {} })
-
-    expect(result).toMatchObject({
-      fields: [],
-    })
-    expect(result.isError).toBe(false)
-    expect(result.error).toBeFalsy()
-  })
```

Comment:
Now that empty schemas can't be passed to `createHeadlessForm` anymore I've removed this test. It was not that meaningful anyway.